core/src/test/java/com/graphhopper/GraphHopperTest.java

```diff
@@ -652,10 +659,13 @@ public void testNorthBayreuthBlockedEdges() {
         assertFalse(rsp.hasErrors(), rsp.getErrors().toString());
         assertEquals(510, rsp.getBest().getDistance(), 10);
 
-        // first point is contained in block_area => error
+        // first point is contained in blocked area => error
+        // todonow: what if a (via-?)point is located within the blocked area? so far we got a dedicated error in this
+        // case, but now we get connection-not-found?
         req = new GHRequest(49.979, 11.516, 49.986107, 11.507202).
```

Comment:
I'm not sure what we should do here. Shouldn't we snap outside the blocked area? And should there be a dedicated exception for custom model areas similar to what we did for `block_area`?

Reply:
It looks like for custom model areas we do snap outside the area: [link](https://graphhopper.com/maps/?point=52.468273%2C13.547468&point=52.451589%2C13.524036&profile=car&layer=Omniscale&custom_model=%7B%22priority%22%3A%5B%7B%22if%22%3A%22in_berlin_bbox%22%2C%22multiply_by%22%3A%220%22%7D%5D%2C%22areas%22%3A%7B%22berlin_bbox%22%3A%7B%22type%22%3A%22Feature%22%2C%22properties%22%3A%7B%7D%2C%22geometry%22%3A%7B%22type%22%3A%22Polygon%22%2C%22coordinates%22%3A%5B%5B%5B13.253%2C52.608%5D%2C%5B13.228%2C52.437%5D%2C%5B13.579%2C52.447%5D%2C%5B13.563%2C52.609%5D%2C%5B13.253%2C52.608%5D%5D%5D%7D%7D%7D%7D)

Unless the points are too far from the area border in which case we get 'Point not found'. So we can leave this as is here.

Reply:
But for some reason in this case we get connection not found?! This also happens on [GH Maps](https://graphhopper.com/maps/?point=49.97958%2C11.520549&point=49.981443%2C11.517759&profile=car&layer=OpenStreetMap&custom_model=%7B%22distance_influence%22%3A15%2C%22speed%22%3A%5B%5D%2C%22priority%22%3A%5B%7B%22if%22%3A%22in_my_area%22%2C%22multiply_by%22%3A%220%22%7D%5D%2C%22areas%22%3A%7B%22my_area%22%3A%7B%22type%22%3A%22Feature%22%2C%22geometry%22%3A%7B%22type%22%3A%22Polygon%22%2C%22coordinates%22%3A%5B%5B%5B11.515818%2C49.981875%5D%2C%5B11.515818%2C49.979522%5D%2C%5B11.521407%2C49.979522%5D%2C%5B11.521407%2C49.981875%5D%2C%5B11.515818%2C49.981875%5D%5D%5D%7D%7D%7D%7D)